test/benchee/benchmark/runner_test.exs

```diff
@@ -1,6 +1,7 @@
 defmodule Benchee.Benchmark.RunnerTest do
   use ExUnit.Case, async: true
   import Benchee.TestHelpers
+  import ExUnit.CaptureIO
```

Comment:
Did you mean to alias this instead of import it?

Reply:
The reason I asked is because on line 157 you're calling `CaptureIO.capture_io` 😉 

Sorry, should have been more clear about that.

Reply:
yeah it's very likely I forgot to `git add` my last changes... now on which of my 3 computers did I do these changes :thinking: